packages/babel-plugin-transform-es2015-modules-umd/src/index.js

```diff
@@ -62,9 +63,8 @@ export default function ({ types: t }) {
             } else if (arg.value === "exports") {
               return t.memberExpression(t.identifier("mod"), t.identifier("exports"));
             } else {
-              return t.memberExpression(t.identifier("global"), t.identifier(
-                t.toIdentifier(basename(arg.value, extname(arg.value)))
-              ));
+              let identifier = _path.basename(arg.value, _path.extname(arg.value));
+              return t.memberExpression(t.identifier("global"), browserGlobals[identifier] ? t.identifier(t.toIdentifier(browserGlobals[identifier])) : t.identifier(t.toIdentifier(identifier)));
```

Comment:
Can you split this up over multiple lines with actual if statements to make it more readable?